src/pykka/_proxy.py

```diff
@@ -14,7 +14,7 @@ class AttrInfo(NamedTuple):
     traversable: bool
 
 
-class ActorProxy:
+class ActorProxyMessageBuilder:
```

Comment:
Documentation still refers to `ActorProxy`, not the `ActorProxyMessageBuilder`. I would also try and explain how this fits in the "architecture" so new users don't have to reverse engineer this from the implementation.

Reply:
New contributors, rather. Users shouldn't need to care about this.